_inc/client/components/jetpack-termination-dialog/features.jsx

```diff
@@ -0,0 +1,150 @@
+/**
+ * External dependencies
+ */
+import { translate as __ } from 'i18n-calypso';
+import Card from 'components/card';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+
+import SingleFeature from './single-feature';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const JetpackTerminationDialogFeatures = ( { purpose, siteBenefits, siteName } ) => {
+	const siteBenefitCount = siteBenefits.length;
+
+	return (
+		<div className="jetpack-termination-dialog__features">
+			<Card>
+				<p className="jetpack-termination-dialog__info">
+					{ purpose === 'disconnect'
+						? __(
+								'Jetpack is currently powering several features of %(siteName)s. Once you disconnect Jetpack, these features will no longer be available and your site may no longer function the same way.',
+								{
+									args: {
+										siteName,
+									},
+								}
+						  )
+						: __(
+								'Jetpack is currently powering several features of %(siteName)s. Once you disable Jetpack, these features will no longer be available and your site may no longer function the same way.',
```

Comment:
"several" is not perfect when you're only using a couple of modules. Do you think we could reword that, or make that part of the string dynamic?

Reply:
going to "Jetpack is powering features on your site" to be more generic, there are a lot of cases going on.